modules/preview-react/text/lib/Label.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
+import {Text, TextProps} from '@workday/canvas-kit-preview-react/text';
+
+export interface TypeLabelProps extends TextProps {
+  hasPointerCursor?: boolean;
```

Comment:
what do you think about adding `cursor` to `text props` ? this way we can drop this prop and we only have disabled?

Reply:
Why `cursor` should be in text props? Do you have any idea of usage case for this?

Reply:
I'm also curious if it's reasonable to add `isRequired` prop to  `Label` component. What do you think?

Reply:
I could see this `Label` Component being used in our form field components. If I look at `FormField.Label` from preview react, it does have a `required` prop which i would add. I think we should remove `disabled` and `hasPointerCursor` props. Those seems like implementation detail that the consumer can add themselves but if you look at w3schools for label, disable isn't a valid attribute, so let's keep this component as simple as possible.